extensions/postgres/tests/introspection/mod.rs

```diff
@@ -1271,11 +1271,11 @@ async fn table_with_generated_always_identity_primary_key() {
       """
       The cursor of the first item in the page
       """
-      startCursor: String! @shareable
+      startCursor: String @shareable
```

Comment:
Just applying `@shareable` to the type is enough, don't need to add it to every field, or it's a composition bug.